src/main/java/com/archisacadeny/course/CourseRepository.java

```diff
@@ -172,7 +172,69 @@ public static int getCourseWithMostStudents() {
         return courseId;
     }
 
+    public static int[] getStudentEnrolledCourses(int studentId) {
+        // RETURNS COURSE ID s FOR NOW
+        String query = "SELECT course_id FROM \"course_student_mapper\" " +
+                "WHERE student_id = '" + studentId + "'" +
+                "GROUP BY course_id ";
 
+        int[] ids;
+        int count = 0;
+
+        try (PreparedStatement statement = DataBaseConnectorConfig.getConnection().prepareStatement(query, ResultSet.TYPE_SCROLL_SENSITIVE,
+                ResultSet.CONCUR_UPDATABLE)) {
+            statement.execute();
+            ResultSet rs = statement.getResultSet();
+
+            rs.last();
+            int numRows = rs.getRow();
+            ids = new int[numRows];
+            rs.beforeFirst();
+
+            while (rs.next()) {
+                ids[count] = rs.getInt("course_id");
+                count++;
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        return ids;
+    }
+
+    public static void createCourseStudentTable() {
+        try (Statement statement = DataBaseConnectorConfig.getConnection().createStatement()) {
+            // SQL query for creating courseStudent table
+            String query = """
+                     DROP SEQUENCE IF EXISTS s_c_mapper_id;
+                     CREATE SEQUENCE s_c_mapper_id INCREMENT BY 1 MINVALUE 0 MAXVALUE 2147483647 START 1;
+                     CREATE TABLE IF NOT EXISTS course_student_mapper (
+                            id INTEGER DEFAULT nextval('s_c_mapper_id') PRIMARY KEY NOT NULL,
+                            course_id INTEGER NOT NULL,
+                            student_id INTEGER NOT NULL,
+                    CONSTRAINT fk_course_id FOREIGN KEY (course_id) REFERENCES "public"."courses"(id),
+                    CONSTRAINT fk_student_id FOREIGN KEY (student_id) REFERENCES "public"."persons"(id)
+                    );
+                    """;
+            statement.execute(query);
+            System.out.println("CourseStudent table has been created in the database..");
+
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void saveCS(int studentID,int courseId){
```

Comment:
this method should be inside this class.
it should be in different package and different class.